config/database.yml

```diff
@@ -20,7 +20,7 @@ default: &default
 
 development:
   <<: *default
-  database: samples_extraction_development
+  database: <%= ENV.fetch('DBNAME','uat_se_dump') %>
```

Comment:
@JamesGlover is this a hangover from your testing?

Reply:
Bleh, yes. Looks like I've already rolled it back locally, must have forgotten to push it up after I fixed it.